Tests/Functional/Action/Platform/Service/OAuth2AccessTokenCreationActionTest.php

```diff
@@ -101,6 +113,11 @@ public function testWithoutGrantType(): void
         );
 
         $this->assertEquals(Response::HTTP_BAD_REQUEST, $this->client->getResponse()->getStatusCode());
+
+        $this->assertHasLogRecord(
+            'OAuth2AccessTokenCreationAction: The authorization grant type is not supported by the authorization server.',
```

Comment:
spoiler: wont fix the too long line, it's a test, and would make it less readable :)